tests/end-to-end/api/09-rooms.js

```diff
@@ -933,4 +934,46 @@ describe('[Rooms]', function() {
 				.end(done);
 		});
 	});
+
+	describe.only('update group dms name', () => {
+		before(async () => {
+			const testUser = await createUser();
+			const admin = 'rocketchat.internal.admin.test';
+			const rocketcat = 'rocket.cat';
+			const usernames = [testUser.username, admin, rocketcat].join(',');
+
+			request.post(api('dm.create'))
+				.set(credentials)
+				.send({
+					usernames,
+				})
+				.end((err, res) => {
+					this.roomId = res.body.room.rid;
+					this.testUser = testUser;
+				});
+		});
+
+		it('should update group name', (done) => {
```

Comment:
can you please add a new test verifying if the DM updates when you change the user's name as well? You'll probably have to update the setting `UI_Use_Real_Name` before running the test:

something like what is done here: https://github.com/RocketChat/Rocket.Chat/blob/d666548e0c78a9d5dbaebe206e55641ae5df004b/tests/end-to-end/api/01-users.js#L999

Reply:
it is also probably better to use `subscriptions.getOne` to verify if the rooms name was correctly updated